armi/reactor/converters/uniformMesh.py

```diff
@@ -123,15 +123,29 @@ def generateCommonMesh(self):
             :id: I_ARMI_UMC_NON_UNIFORM
             :implements: R_ARMI_UMC_NON_UNIFORM
 
+            Generates a core-wide mesh that preserves important material boundaries, in
+            particular the terminations of the fuel column as well as control assembly
+            material. Calls ``_computeAverageAxialMesh``, which operates by first
+            collecting all the mesh points for every assembly (``allMeshes``) and then
+            averaging them together using ``average1DWithinTolerance`` to obtain the
+            core-wide mesh.
```

Comment:
I don't think any of this text needs changing, but I think maybe both of the `impl` tags here really belong in `_decuspAxialMesh`. Both functions ("preserve material boundaries" and "minimumMeshSize") are happening inside `_decuspAxialMesh`, and won't happen at all unless `minimumMeshSize is not None` so that `_decuspAxialMesh` is called.

Reply:
I guess we should also append this text to clarify that the material boundary preservation will only occur if `minimumMeshSize` is given.

Reply:
b7b9c0143f73bdf47abb6293d771b1d458143e06 let me know what you think